src/modules/sub-items/components/pagination/pagination.btn.component.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const PaginationBtnComponent = ({ label, disabled, additionalClasses, onPageChange, page }) => {
+    const handleClick = () => {
+        if (!disabled && Number.isInteger(page)) {
+            onPageChange(page);
+        }
+    };
+
+    let className = `page-item ${additionalClasses}`;
+
+    className = disabled ? `${className} disabled` : className;
```

Comment:
Is the `disabled` class coming from Bootstrap? Why not to add `disabled` attribute on the button?

Reply:
Styling does not work if I switch to attribute.